docs/examples.rst

```diff
@@ -122,15 +139,36 @@ Web3 can help you convert between denominations.  The following denominations ar
 | tether       | 1000000000000000000000000000000 |
 +--------------+---------------------------------+
 
+Picking up from the previous example, the largest account contained
+3841357360894980500000001 wei. You can use the :meth:`~web3.fromWei` method
+to convert that balance to ether (or another denomination).
+
+.. code-block:: python
+
+    >>> web3.fromWei(3841357360894980500000001, 'ether')
+    Decimal('3841357.360894980500000001')
+
+To convert back to wei, you can use the inverse function, :meth:`~web3.toWei`.
+Note that Python's default floating point precision is insufficient for this
+use case, so it's necessary to cast the value to a
+`Decimal <https://docs.python.org/3/library/decimal.html>`_ if it isn't already.
+
+.. code-block:: python
+
+    >>> from decimal import Decimal
+    >>> web3.toWei(Decimal('3841357.360894980500000001'), 'ether')
+    3841357360894980500000001
+
+Best practice: If you need to work with multiple currency denominations, default
```

Comment:
I would probably put that into a `.. note:` but no strong feelings.

Reply:
I like the idea, but I'll push back, because I'd want to include the code sample that follows. No other `Note`s in the docs include a code snippet and I think it'd make for an overly large aside.